pkg/cache/tas_cache_test.go

```diff
@@ -2119,3 +2120,143 @@ func TestFindTopologyAssignment(t *testing.T) {
 		})
 	}
 }
+
+func TestFindTopologyAssignmentForTwoPodSets(t *testing.T) {
```

Comment:
I would probably prefer to make the existing test more generic for handling multiple PodSets, because "two PodSets" feels quite constraining either.

Reply:
I agree. I would also like to make already existing tests more generic. Since we want to fix this bug before release I wanted to do something as simple as possible without any risk. I am more than happy to cleanup it after release to make existing tests more versatile.